docs/Antora/modules/howto_guides/pages/howto_configure_instance.adoc

```diff
@@ -0,0 +1,291 @@
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+int main(int argc, const char* argv[])
+{
+    daq::InstancePtr instance = daq::Instance();
+    daq::DevicePtr device = instance.addDevice("daq.opcua://127.0.0.1/");
+
+    return 0;
+}
+----
+====
+
+== Creating openDAQ(TM) Instance with Instance Builder
+
+openDAQ(TM) provides an advanced way to configure an Instance using the Instance Builder. With the Instance Builder, developers can configure the Logger, Scheduler, Module Manager, and Root Device by setting custom modules or modifying default objects.
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+int main(int argc, const char* argv[])
+{
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder()
+    daq::InstancePtr instance = instanceBuilder.build();
+    daq::DevicePtr device = instance.addDevice("daq.opcua://127.0.0.1/");
+
+    return 0;
+}
+----
+====
+
+== Configure openDAQ(TM) Instance Logger
+
+By default, the Instance Logger level is set to the info log level if the release version of the project was built or debug if the project was built in debug mode. There are two ways to override the default level. The first one is to set the global default logger level.
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+int main(int argc, const char* argv[])
+{
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder();
+    instanceBuilder.setGlobalLogLevel(daq::LogLevel::Warn);
+    daq::InstancePtr instance = instanceBuilder.build();
+
+    return 0;
+}
+----
+====
+
+In addition to changing the logging level, the developer can also set the logging level for individual components and sinks. In the example below, the developer sets two logger sinks: `StdOutLoggerSink` with the default log level and `BasicFileLoggerSink` with the error log level. Additionally, custom log levels are set for the components `Instance`` and `Some Module`.
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+int main(int argc, const char* argv[])
+{
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder();
+                        .addLoggerSink(StdOutLoggerSink())
+                        .setSinkLogLevel(BasicFileLoggerSink("logfile.log"),  daq::LogLevel::Error);
+                        .setComponentLogLevel("Instance", daq::LogLevel::Warn);
+                        .setComponentLogLevel("Some Module", daq::LogLevel::Off);
+
+    daq::InstancePtr instance = instanceBuilder.build();
+
+    return 0;
+}
+----
+====
+
+An advanced way of configuring the Instance logger is to configure the logger object and set it with the `setLogger` method.
```

Comment:
I'd try to avoid the mention of language-specific function call notation, as it becomes harder to write documentation for other languages this way. I'd reference function calls only in  code samples.